contrib_bots/lib/poll.py

```diff
@@ -0,0 +1,105 @@
+
+
+class PollHandler(object):
+    '''
+    This plugin facilitates polling.
+    '''
+
+    def __init__(self):
+        # polls is a dict of dict. First dict corresponds to poll title,
+        # second refers to answers in poll
+        self.polls = dict()
+
+        # streams is a convenience object that keeps track of the latest
+        # question for each stream.
+        self.streams = dict()
+
+    def usage(self):
+        return '''
+        This plugin facilitates polling.
+
+        Commands:
+        @question poll title: option 1, option 2, option 3
+        A valid poll must use a colon (:) to devide question from answers.
```

Comment:
A typo - `devide` instead of `divide`.

Also, I don't think `poll title` is the right name for what this bot is doing - it's simply gathering answers to questions (unless I'm wrong?). I'd thus call it `question` instead of `poll title` everywhere, since the `poll title` can suggest a poll can have multiple questions (that's what I'd expect from a poll).